plugins/hive-plugin/src/test/java/edu/berkeley/ground/plugins/hive/TestGroundMetastore.java

```diff
@@ -0,0 +1,224 @@
+            return false;
+        }
+
+        @Override
+        public FileMetadataExprType getMetadataType(String inputFormat) {
+            return null;
+        }
+
+        @Override
+        public SearchArgument createSarg(byte[] expr) {
+            return null;
+        }
+
+        @Override
+        public FileFormatProxy getFileFormatProxy(FileMetadataExprType type) {
+            return null;
+        }
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        HiveConf conf = new HiveConf();
+        Deadline.registerIfNot(100000);
+        conf.setVar(HiveConf.ConfVars.METASTORE_EXPRESSION_PROXY_CLASS, MockPartitionExpressionProxy.class.getName());
+        HiveConf.setVar(conf, HiveConf.ConfVars.METASTORE_CONNECTION_DRIVER, "test_connection");
+        conf.set(GRAPHFACTORY_CLASS, PostgresGraphVersionFactory.class.getName());
+        conf.set(NODEFACTORY_CLASS, PostgresNodeVersionFactory.class.getName());
+        conf.set(EDGEFACTORY_CLASS, PostgresEdgeVersionFactory.class.getName());
+        GroundReadWrite.setConf(conf);
+        groundStore = new GroundStore();
+        groundStore.setConf(conf);
+        // dropAllStoreObjects(groundStore);
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    /**
+     * Test database operations
+     */
+    @Test
+    // @Ignore
+    public void testDatabaseOps() throws MetaException, InvalidObjectException, NoSuchObjectException {
+        int numDBs = groundStore.getAllDatabases().size();
+        Database db1 = new Database(DB1, "description", "locationurl", new HashMap<String, String>());
+        Database db2 = new Database(DB2, "description", "locationurl", new HashMap<String, String>());
+        groundStore.createDatabase(db1);
+        numDBs++;
+        groundStore.createDatabase(db2);
+        numDBs++;
+
+        String dbName = groundStore.getDatabase(DB1).getName();
+        Assert.assertEquals(DB1, dbName);
+        dbName = groundStore.getDatabase(DB2).getName();
+        Assert.assertEquals(DB2, dbName);
+        List<String> databases = groundStore.getAllDatabases();
+        Assert.assertEquals(numDBs, databases.size());
+        for (String database : databases) {
+            System.out.println("database: " + database);
+        }
+        Assert.assertTrue(databases.contains(DB2));
+        Assert.assertTrue(databases.contains(DB1));
+        Assert.assertEquals(true, groundStore.dropDatabase(DB1));
+        numDBs--;
+
+        List<String> databases2 = groundStore.getAllDatabases();
+        Assert.assertFalse(databases2.contains(DB1));
+        Assert.assertEquals(numDBs, databases2.size());
+
+        Database db2v2 = new Database(DB2, "new description", "another_locationurl", new HashMap<String, String>());
+        groundStore.alterDatabase(DB2, db2v2);
+        Database db2v2r = groundStore.getDatabase(DB2);
+        Assert.assertEquals(db2v2.getName(), db2v2r.getName());
```

Comment:
Can we `import static org.junit.Assert.*` and not prefix everything with `Assert.`, please?